src/bio.c

```diff
@@ -66,13 +66,16 @@ bio_tx(fido_dev_t *dev, uint8_t subcmd, cbor_item_t **sub_argv, size_t sub_argc,
 	fido_blob_t	*ecdh = NULL;
 	fido_blob_t	 f;
 	fido_blob_t	 hmac;
-	const uint8_t	 cmd = CTAP_CBOR_BIO_ENROLL_PRE;
+	uint8_t		 cmd;
```

Comment:
consider moving the inspection of dev to a local subroutine (e.g. get_cmd() or get_cmd_byte()) and keeping the variable const (same for credman.c)

Reply:
thanks!